auslib/test/blobs/test_apprelease.py

```diff
@@ -104,6 +104,143 @@ def setUp(self):
+                "partial": "j1-partial.mar",
+                "complete": "complete.mar"
+            },
+            "platforms": {
+                "p": {
+                    "buildID": 30,
+                    "OS_FTP": "o",
+                    "OS_BOUNCER": "o",
+                    "locales": {
+                        "l": {
+                            "partial": {
+                                "filesize": 6,
+                                "from": "samplePartial1",
+                                "hashValue": "5"
+                            },
+                            "complete": {
+                                "filesize": 38,
+                                "from": "*",
+                                "hashValue": "34"
+                            }
+                        },
+                        "en-US": {
+                            "partial": {
+                                "filesize": 6,
+                                "from": "samplePartial2",
+                                "hashValue": "5"
+                            },
+                            "complete": {
+                                "filesize": 38,
+                                "from": "*",
+                                "hashValue": "34"
+                            }
+                        }
+                    }
+                },
+                "p2": {
+                    "buildID": 31,
+                    "OS_FTP": "o",
+                    "OS_BOUNCER": "o",
+                    "locales": {
+                        "l": {
+                            "partial": {
+                                "filesize": 6,
+                                "from": "samplePartial3",
+                                "hashValue": "5"
+                            },
+                            "complete": {
+                                "filesize": 38,
+                                "from": "*",
+                                "hashValue": "34"
+                            }
+                        },
+                        "en-GB": {
+                            "partial": {
+                                "filesize": 6,
+                                "from": "samplePartial4",
+                                "hashValue": "5"
+                            },
+                            "complete": {
+                                "filesize": 38,
+                                "from": "*",
+                                "hashValue": "34"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        """)
+
+    def testGetPartialReleaseReferences_Happy_Case(self):
+        partial_releases = self.sampleReleaseBlob.getReferencedReleases()
+        self.assertTrue(4, len(partial_releases))
+        for item in (['samplePartial1', 'samplePartial2', 'samplePartial3', 'samplePartial4']):
+            self.assertTrue(item in partial_releases)
```

Comment:
This test is correct, but the asserts could be simplified as:
self.assertTrue(partial_releases, ["samplePartial1", "samplePartial2", "samplePartial3", samplePartial4"])